src/pages/mesh/basic/create-mesh.md

```diff
@@ -111,7 +111,7 @@ The edge mesh URL offers several benefits because the edge is closer to your dat
 
 <InlineAlert variant="warning" slots="text"/>
 
-Edge meshes do not currently support Hooks or SOAP handlers. If you need to use these features, you must use a legacy mesh. These features will be available in edge meshes in the future.
+Edge meshes do not currently support SOAP handlers. If you need to use this feature, you must use a legacy mesh. SOAP handler support will be available in edge meshes in the future.
```

Comment:
I believe we're actually removing SOAP handler support entirely on edge mesh. Not sure what message to convey here.